### ISSUE TITLE
Error: package or namespace load failed for 'clusterProfiler': object 'get_fun_from_pkg' is not exported by 'namespace:rvcheck'is:issue is:open 

### ISSUE THREAD

**Issue by XiaohanLuDuke** [YuLab-SMU/clusterProfiler#374]
### Prerequisites

+ [ ] Have you read [Feedback](https://guangchuangyu.github.io/clusterprofiler/#feedback) and follow the [guide](https://guangchuangyu.github.io/2016/07/how-to-bug-author/)?
	* [ ] make sure your are using the latest release version
	* [ ] read the [documents](https://guangchuangyu.github.io/clusterprofiler/documentation/)
	* [ ] google your quesion/issue

### Describe you issue

* [ ] Make a reproducible example (*e.g.* [1](https://gist.github.com/talonsensei/e1fad082657054207f249ec98f0920eb))
* [ ] your code should contain comments to describe the problem (*e.g.* what expected and actually happened?)

### Ask in right place

* [ ] for bugs or feature requests, post here (github issue)
* [ ] for questions, please post to [Bioconductor](https://support.bioconductor.org/) or [Biostars](https://www.biostars.org/) with tag `clusterProfiler`

**Comment by XiaohanLuDuke** [comment]
When I library (ClusterProfiler), the error message show up as below:
Error: package or namespace load failed for 'clusterProfiler': object 'get_fun_from_pkg' is not exported by 'namespace:rvcheck'is:issue is:open 
I there anything wrong?

**Comment by zamlerd** [comment]
I am getting the same error

Error: package or namespace load failed for ‘clusterProfiler’:
 object ‘get_fun_from_pkg’ is not exported by 'namespace:rvcheck'

**Comment by huerqiang** [comment]
@zamlerd @XiaohanLuDuke Please see #371

**Comment by XiaohanLuDuke** [comment]
> @zamlerd @XiaohanLuDuke Please see #371

Thanks! I am using the latest Rstudio and the packages was all install.  But still found this error.

**Comment by XiaohanLuDuke** [comment]
> @zamlerd @XiaohanLuDuke Please see #371

I will try！

**Comment by huerqiang** [comment]
@XiaohanLuDuke  You should update your R(not just Rstudio) and reinstall `clusterProfiler`.

**Comment by torgheh** [comment]
I had the same issue. I think it is caused by an update in the `rvcheck` package. They've removed and renamed all the methods without releasing a new major version.
 
In my case I can not update R as I am not an administrator so I downgraded the `rvcheck`. 
```R
install.packages('devtools')
require(devtools)
install_version("rvcheck", version = "0.1.8", repos = "http://cran.us.r-project.org")
library(rvcheck)
```

**Comment by musaidris** [comment]
same here .. happened after updating many packages ..

`library(clusterProfiler)`
```
Error: package or namespace load failed for ‘clusterProfiler’:
 object ‘get_fun_from_pkg’ is not exported by 'namespace:rvcheck'
In addition: Warning message:
package ‘clusterProfiler’ was built under R version 4.0.2 
```

installing `ggtree `and `yulab.utils` didn't solve the problem, 
problem also not solved by: 

`if(!requireNamespace("remotes", quietly = TRUE))
    install.packages("remotes")
remotes::install_github("YuLab-SMU/clusterProfiler")`


`sessionInfo()`
```
R version 4.0.1 (2020-06-06)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 10 x64 (build 19043)

Matrix products: default

locale:
[1] LC_COLLATE=English_Netherlands.1252  LC_CTYPE=English_Netherlands.1252    LC_MONETARY=English_Netherlands.1252
[4] LC_NUMERIC=C                         LC_TIME=English_Netherlands.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
  [1] enrichplot_1.8.1     bit64_4.0.5          RColorBrewer_1.1-2   progress_1.2.2       httr_1.4.2          
  [6] tools_4.0.1          utf8_1.2.2           R6_2.5.1             DBI_1.1.1            BiocGenerics_0.34.0 
 [11] colorspace_2.0-2     prettyunits_1.1.1    tidyselect_1.1.1     gridExtra_2.3        bit_4.0.4           
 [16] compiler_4.0.1       Biobase_2.48.0       scatterpie_0.1.7     xml2_1.3.2           triebeard_0.3.0     
 [21] scales_1.1.1         ggridges_0.5.3       yulab.utils_0.0.2    stringr_1.4.0        digest_0.6.28       
 [26] rmarkdown_2.11       DOSE_3.14.0          pkgconfig_2.0.3      htmltools_0.5.2      fastmap_1.1.0       
 [31] rlang_0.4.11         readxl_1.3.1         RSQLite_2.2.8        gridGraphics_0.5-1   generics_0.1.0      
 [36] farver_2.1.0         jsonlite_1.7.2       BiocParallel_1.22.0  GOSemSim_2.14.2      dplyr_1.0.7         
 [41] magrittr_2.0.1       ggplotify_0.1.0      GO.db_3.11.4         Matrix_1.2-18        Rcpp_1.0.7          
 [46] munsell_0.5.0        S4Vectors_0.26.1     fansi_0.5.0          viridis_0.6.1        lifecycle_1.0.1     
 [51] stringi_1.7.4        yaml_2.2.1           ggraph_2.0.5         MASS_7.3-54          plyr_1.8.6          
 [56] qvalue_2.22.0        grid_4.0.1           blob_1.2.2           parallel_4.0.1       ggrepel_0.9.1       
 [61] DO.db_2.9            crayon_1.4.1         lattice_0.20-41      graphlayouts_0.7.1   cowplot_1.1.1       
 [66] splines_4.0.1        hms_1.1.1            knitr_1.34           pillar_1.6.3         fgsea_1.14.0        
 [71] igraph_1.2.6         reshape2_1.4.4       stats4_4.0.1         fastmatch_1.1-3      glue_1.4.2          
 [76] evaluate_0.14        ggfun_0.0.4          downloader_0.4       BiocManager_1.30.16  data.table_1.14.0   
 [81] urltools_1.7.3       vctrs_0.3.8          tweenr_1.0.2         cellranger_1.1.0     gtable_0.3.0        
 [86] purrr_0.3.4          polyclip_1.10-0      tidyr_1.1.3          assertthat_0.2.1     cachem_1.0.6        
 [91] ggplot2_3.3.5        xfun_0.26            ggforce_0.3.3        europepmc_0.4.1      tidygraph_1.2.0     
 [96] viridisLite_0.4.0    tibble_3.1.4         rvcheck_0.2.0        tinytex_0.33         AnnotationDbi_1.52.0
[101] memoise_2.0.0        IRanges_2.22.2       ellipsis_0.3.2 
```     
>

**Comment by musaidris** [comment]
update: 
problem not solved by updating everything, on R version 4.1.1 (2021-08-10) -- "Kick Things"
Problem solved only by downgrading rvcheck as suggested by Torgheh, thx.

**Comment by GuangchuangYu** [comment]
see <https://github.com/YuLab-SMU/clusterProfiler/issues/371#issuecomment-921882474>
